bundle/manifests/submariner.clusterserviceversion.yaml

```diff
@@ -724,6 +724,33 @@ spec:
               terminationGracePeriodSeconds: 10
       permissions:
       - rules:
+        - apiGroups:
+          - submariner.io
+          resources:
+          - brokers
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - delete
+        - apiGroups:
+          - submariner.io
+          resources:
+          - brokers
+          verbs:
+          - get
+          - list
+          - watch
```

Comment:
@tpantelis The bundle job asks me to do the changes in this way , but the duplicated broker roles needs to be in different namespaces ( broker and operator ) any idea how we could resolve it?

Reply:
Removed the broker one , but addon-agent needs permission right ? does that come from this file ?

Reply:
I don't think so. The u/s broker controller doesn't run in the managed cluster (since in ACM we don't create a Broker instance).